etc/institutions-auth.xsl

```diff
@@ -36,10 +36,12 @@
                         <id>esu</id>
                         <user>
                             <!--  Each institution has its customized mapping of attributes  -->
-                            <username><xsl:value-of select="//attribute[@name='eppn']/@value"/></username>
+                            <username><xsl:value-of select="//attribute[@name='mail']/@value"/></username>
```

Comment:
This is a change that is not related to the institution department update. I slipped it into the PR because it is a trivial change and it is just an example.

* For the email attribute, we have moved from using `eppn` to `mail`.
* `eppn` is used for user's institutional identity only. Its value could be the same as the value of `mail`. It could be of the email format but is not an actual email. It could be just a string. Thus, it is strongly recommended to use `mail` (or other dedicated attributes for email) instead of using `eppn` for OSF `username`.
* `mail` is one of the defaults attributes released to InCommon R&S SPs by R&S IdPs.

cc @felliott thanks for asking about this change, I added this post merge comment for future references